mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS kp net/minecraft/util/Identifier
-	FIELD a owner Ljava/lang/String;
+	FIELD a namespace Ljava/lang/String;
```

Comment:
As I said in IRC before, I don't really like `owner` because it (in my opinion) implies that there's some object backing ownership. `namespace` is more commonly strictly a key, which is true in `Identifier`.